schema.py

```diff
@@ -91,10 +94,14 @@ def resolve_regions(self, info):
         region_names = set([s.region for s in securities])
         return [BookRegion(name=region_name, book_name=self.name, region_name=region_name) for region_name in region_names]
 
+    def resolve_region(self, info, name):
+        return BookRegion(name=name, book_name=self.name, region_name=name)
```

Comment:
Are we using the `name` for anything in `BookRegion`? These constructors are a bit confusing with all the `name`s being passed around.

Reply:
It is there because if it weren't, we wouldn't be able to ```
Book {
  region {
    name
  }
}
```

Reply:
(I think you are the one that figured this out)

Reply:
Ah, yeah. Outsmarted myself on that one.